tests/unit/core/utils/test_server.js

```diff
@@ -0,0 +1,90 @@
+import path from 'path';
+
+import fs from 'fs-extra';
+import MockExpressResponse from 'mock-express-response';
+
+import {
+  getFeatureFlagName,
+  viewFrontendVersionHandler,
+} from 'core/utils/server';
+import { getFakeConfig } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  describe('viewFrontendVersionHandler', () => {
+    const basePath = path.join(__dirname, '__fixtures__');
```

Comment:
nit: I don't think `__fixtures__` needs so many underscores here because there is no chance of naming collisions.

Reply:
works for me (old habit, sorry)